interface/main/left_nav.php

```diff
@@ -725,55 +719,6 @@ function reloadEncounter(frname) {
   }
  }
 
- // Clear and reload issue-related menu items for active_pid.
- // Currently this only applies to athletic teams, but might be implemented
```

Comment:
Given the comment I would leave these functions here but comment them out, as they may be of interest later.